utils.py

```diff
@@ -491,15 +491,26 @@ def hasDockerLibrary(step):
     builderName = str(step.getProperty("buildername"))
 
     # from https://github.com/MariaDB/mariadb-docker/blob/next/update.sh#L7-L15
-    if fnmatch.fnmatch(branch, "10.[4-6]"):
+    if fnmatch.fnmatch(branch, "10.[56]"):
         dockerbase = "ubuntu-2004-deb-autobake"
     elif fnmatch.fnmatch(branch, "10.11"):
         dockerbase = "ubuntu-2204-deb-autobake"
-    elif fnmatch.fnmatch(branch, "11.[012]"):
+    elif fnmatch.fnmatch(branch, "11.[12]"):
         dockerbase = "ubuntu-2204-deb-autobake"
     else:
         dockerbase = "ubuntu-2404-deb-autobake"
 
+    # UBI images
+    if branch in [
+        "10.6",
+        "10.11",
+        "11.4",
+        "11.5",
+        "11.6",
+        "main",
+    ] and builderName.endswith("amd64-rhel-9-rpm-autobake"):
```

Comment:
There is a bit of a difference between PROD & DEV.
On PROD:
amd64-rhel-9-rpm-autobake will trigger amd64-rhel8-dockerlibrary builder which runs on bb-rhel8-docker
On DEV
amd64-rhel-9-rpm-autobake will trigger amd64-rhel9-dockerlibrary which runs on bb-rhel9-docker.

This is due to the fact of how non-latent builders / workers are generated currently for docker-library.
Is this considered acceptable?